bots/discord/game-server-watcher/README.md

```diff
@@ -16,3 +16,7 @@ Refer to the wiki on how to acquire tokens for:
 ## Server Ports
 GSW Control Panel requires one port for http traffic.  
 (It can be any port.)  
+
+| Port    | default |
+|---------|---------|
+| Web     | 8080    |
```

Comment:
8080 is wings port so that is not posible

Reply:
that's just the default value within the app if the PORT env var is not set.. which should not happen in pterodactyl unless someone edits the startup config if i'm not mistaken.

anyway how about i edit it like this:

| Port    | suggested |
|---------|-----------|
| Web     | 58080     |

or do you suggest i add other safeguards? if somehow the PORT defaults back to 8080 the app/server will just fail to start anyway since the port won't be able to bind, but the user can't allocate the wings port, so again it should not be an issue.

another way to approach this is to add the PORT env to the variables, and make the user choose/set a pre-selected port, but to me this seemed way more involved than just using whatever port the user is choosing/allowed to allocate to the server.

any best practice feedback on this is much appreciated

Reply:
The default is just an example. There is no way that they can bind to 8080 as it uses primary port. You can fine just set 8081. The number just can not be 8080 or 2022